### ISSUE TITLE
您好，请教一个关于few-shot实验的细节问题。

### ISSUE THREAD

**Issue by Haitao3323** [thunlp/Few-NERD#33]
在您的论文中，将整个实体集划分成了三个部分，用作训练、验证和测试，并且进行了实体的替换以防止信息泄露。论文中是这么描述的：To avoid the observation of new entity types in the training phase, we replace the labels of entities that belong to Etest with O in the training set. Similarly, in the test set, entities that belongs to Etrain and Edev are also replaced by O. 根据这一描述，训练集只将属于测试集的实体替换成了O，属于验证集的实体似乎是没有被替换的，并且也没有提到验证集的处理方法。我下载了您公开的intra数据集并写了脚本统计了一下，发现训练集中属于验证集的实体也被替换成了O，而且三个数据集的实体是互不相交的。
所以我现在有点困惑，论文的描述和实际的做法是不一致的，是我对您论文中的描述理解错误了吗？

**Comment by cyl628** [comment]
Hi, sorry for the confusion. As you have found out, under `inter` and `intra` mode, for all the three datasets, we **do** replace the entity types in the other two datasets with `O`, i.e. we ensure that the entity types have **no overlap** among the train, dev, and test split. The description in the paper should be modified.
